fake/src/impls/url/mod.rs

```diff
@@ -0,0 +1,12 @@
+use crate::{Dummy, Fake, Faker};
+use rand::Rng;
+use url::Url;
+
+const FQDN: &str = "https://example.com";
+
+impl Dummy<Faker> for Url {
+    fn dummy_with_rng<R: Rng + ?Sized>(_: &Faker, rng: &mut R) -> Self {
+        let path: String = Faker.fake_with_rng(rng);
```

Comment:
are arbitrary string can be a valid path?

Reply:
I think it is. If path is `[a-zA-Z]`, it must be valid.

However, I think it's better to define paths as const.
FIXED: [c736536](https://github.com/cksac/fake-rs/pull/190/commits/c736536a9b80d557bd52662aab13e1cea2086ea3)